src/index.html

```diff
@@ -1,28 +1,28 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>To-do List</title>
-    <script src="./index.js" defer></script>
-    <link rel="stylesheet" href="src/style.css">
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>To do list</title>
+  </head>
+  <body>
     <main>
-        <section class="to-do-container">
-            <div class="title-and-refresh">
-                <h1>Today's To Do</h1>
-            </div>
-            <div class="input_and_add">
-                <form action="">
-                    <input type="text" class="input-task" placeholder="Add to your list...">
-                </form>
-                <button class="add">Add</button>
-            </div>
-            <ul class="list-div"></ul>
-            <button class="clearButton">Clear all completed tasks</button>
-        </section>
-    </main>
-</body>
-</html>
\ No newline at end of file
+      <section class="to-do-container">
+          <div class="title-and-refresh">
+              <h1>Today's To Do</h1>
+          </div>
+          <div class="input_and_add">
+              <form action="">
+                  <input type="text" class="input-task" id="input-task" placeholder="Add to your list...">
+              </form>
+              <button class="add">Add</button>
+          </div>
+          <ul class="list-div">
+
+          </ul>
+          <button class="clearButton">Clear all completed tasks</button>
```

Comment:
- [ ] Good job so far @kkmanuu There are just a few changes you need to make and you'll be on your way to the next Milestone :+1: 
- [ ] After cloning your project, I came across a series of errors that broke your app, and as a result, I was unable to test the functionalities you were asked to implement in this Milestone which include `Adding a todo`, `Deleting a todo`, and `editing a todo/ updating a todo` among others as seen in the [Project Requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md). :+1: 
- [ ] So I will suggest that you fix all the errors in your project so the app can run and look as expected so it can be tested properly to ensure that all the functionalities and project requirements have been met dutifully :wink:  
- Error message 
![image](https://github.com/kkmanuu/To-Do-Lists/assets/102699958/bba52da6-492e-4eb0-9393-e6a8323fdc69)
- Your Current UI
![image](https://github.com/kkmanuu/To-Do-Lists/assets/102699958/ce6d16c8-dfe7-4285-bb1f-e7de75a54eb1)

Reply:
@NahnahAJ Done